### ISSUE TITLE
[Question] Will this course offer multilingual content?

### ISSUE THREAD

**Issue by LaoshuBaby** [HMUniversity/CS101#1]
Noticed that everything in this course has been written in English currently, but we all know that the vast majority of our students are not from English-speaking regions or as native English speakers. Are there any plans to consider introducing multiple languages for this course in the future?

**Comment by George-Miao** [comment]
Probably Chinese will be the easiest one for most people to get on as the start point of i18n

**Comment by mmdjiji** [comment]
Yes, I think it we can start a second language from Chinese, but mkdocs seems cannot support multi-languages smoothly, so maybe we should transfer it to vitepress first.

**Comment by Cubik65536** [comment]
> Yes, I think it we can start a second language from Chinese, but mkdocs seems cannot support multi-languages smoothly, so maybe we should transfer it to vitepress first.

Starting with Chinese for multilingual project is great, so I think we should start to migrate to VuePress/VitePress/Hexo (or other similar frameworks) as soon as possible. Otherwise, this could be a problem once we have more content written.

**Comment by cupen** [comment]
Well, we want good good study?  Maybe it's not in accordance with the school motto. Haha.  :rofl:

**Comment by afterow** [comment]
> > Yes, I think it we can start a second language from Chinese, but mkdocs seems cannot support multi-languages smoothly, so maybe we should transfer it to vitepress first.
> 
> Starting with Chinese for multilingual project is great, so I think we should start to migrate to VuePress/VitePress/Hexo (or other similar frameworks) as soon as possible. Otherwise, this could be a problem once we have more content written.

I usually use Obsidian, and use [Rookie Tutorial](www.runoob.com) for online learning and [Bilibili](www.bilibili.com) to learn computers.
I have never used VuePress/VitePress/Hexo, do you think there is any easy migration?
